app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java

```diff
@@ -102,6 +102,22 @@ public boolean registerNewPet(User user, Pet pet) {
             }
         });
 
+        executorService.shutdown();
+        try {
+            executorService.awaitTermination(5, TimeUnit.MINUTES);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['5' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5650467)